portfolio/src/main/java/com/google/sps/data/BlogHashMap.java

```diff
@@ -24,44 +24,47 @@
 import java.util.Set;
 
 public final class BlogHashMap {
-  private static final String[] internalTags = {"#general", "#wellbeing", "#music", "#games"};
+  private final String[] INTERNAL_TAGS = {"#general", "#wellbeing", "#music", "#games"};
+  private Map<String, LinkedList<BlogMessage>> map;
```

Comment:
Nit: "map" is okay in this scenario since it should be the only hashmap in BlogHashMap for now. Going forward, it would be good to avoid variable names such as list, map, array, etc. because they are too general, and actual reader will know their they are "map" from their type already.